src/ray/core_worker/reference_count.cc

```diff
@@ -247,12 +247,28 @@ void ReferenceCounter::SetNestedRefInUseRecursive(ReferenceTable::iterator inner
   }
 }
 
+void ReferenceCounter::ReleaseAllLocalReferences() {
+  absl::MutexLock lock(&mutex_);
+  for (auto &ref : object_id_refs_) {
+    /// Remove multiple times.
```

Comment:
This comment is confusing. Can you just remove it or re-comment as "Remove it until owned ref goes to 0?"

Reply:
deleted.